### ISSUE TITLE
Climate Model Explanations

### ISSUE THREAD

**Issue by hadinon** [ncsco/pinemap-dss-help#63]
*Create a tiered approach to the explanations.  The DSS Overview landing page and FAQ box could include very basic information, but add more detailed explanations in tooltips or on separate pages. 
*On the Overview/Introduction page, link to new MACA FAQ website.  Maybe create a slightly separate version with questions tailored to the DSS users? 
(MEDIUM PRIORITY)

**Comment by hadinon** [comment]
Maybe on the "Climate Data" tab of the DSS Overview page, add a bullet to say, "For more details on MACA, please click here" (or something similar) and have "here" linked to: http://climate.ncsu.edu/hadinon/pinemap/MACA_FAQs.php

Would linking to the MACA FAQ site under "Other Resources" provide enough detail about this?

**Comment by daviswx** [comment]
I think linking to the FAQ page would provide sufficient detail.  We also discussed creating a slightly different FAQ page just for the DSS that might focus less on data analysis and more on data interpretation.  I can't remember exactly what we discussed that this might entail, but maybe we can brainstorm some ideas for what DSS users might want to know about the MACA data.

**Comment by daviswx** [comment]
Here are some thoughts about how the FAQ page could be tailored to the DSS:
- **Baseline data tab**
  - Get rid of the final part of the title ("How can I use it in my analysis?").
  - In the second paragraph, change the wording "should be compared" to something like "was compared".
- **Observed data tab**
  - Change the title to something like "Which observed dataset was used for the PINEMAP DSS?".
- **Best practices tab**
  - I'm debating whether it's worth including this tab.  Ultimately, it's probably too much information for DSS users since they aren't directly involved with any of the analysis steps.  Instead of this, we may want a brief explanation (possibly under the future time slices tab) about how the averages, maximums, and minimums for the 20-year time slices were determined (e.g., average daily data for each model, then average and compute stats for all 20 models).
- **Future time slices tab**
  - Change the title to something like "Why are 20-year future time slices used in the PINEMAP DSS?".
- **Model subset tab**
  - Get rid of this for the DSS FAQ page.
- **Citation tab**
  - Either get rid of this, or change it to be a citation for the PINEMAP DSS (rather than the MACA data).
- **Anything else tab**
  - Get rid of the paragraph about the ATP presentation.

I can think of a few things that DSS users might want to know that the modelers wouldn't:
- **What are these 20 different climate models?**  Who runs them?  What makes them different from one another?  Are they independent from one another?
- **Why do we have any confidence in the models' ability to predict future conditions?**  How accurate is the model baseline vs. historical observations?  Is this model error shown anywhere?
- **What do the RCPs actually represent?**  Why does the PINEMAP DSS only show two of them?  How much confidence do we have that future conditions will play out as these RCPs indicate?
- **Is the range of historical conditions wider than the possibilities painted by the model projections?**  In other words, how do I know when the model projections become actionable?
- **Where do the productivity datasets come from?**  We may want an explanation of 3PG/G&Y/WaSSI and how the MACA data is used with them.

In some cases, it would just be a matter of adding extra information, possibly by expanding the paragraphs about GCMs and RCPs in the first tab into their own sections.  In other cases, like with the historical variability one, we might not have that data available at the moment but we may still want to think about writing up an explanation.

Also, if you want, we could put your code for the FAQ page within the DSS template so it matches a bit better.

**Comment by hadinon** [comment]
Add these details about future projections to the FAQ page (and link to FAQ page from the Climate Data tab on the Intro page): 
- Future projections come from a set of global climate models, which aim to simulate the physics and chemistry of our atmosphere, land, and oceans to paint a picture of climate over several decades or longer
- DSS tools are based on the University of Idaho's Multivariate Adaptive Constructed Analogs (MACA) dataset, which connects large-scale output from global climate models to local scales at a sufficient resolution for decision making

**Comment by hadinon** [comment]
The new FAQ site for DSS users is available at:
http://climate.ncsu.edu/hadinon/pinemap/MACA_DSS_FAQs.php
Note: we can change the URL later if necessary and/or put the finalized code into the DSS template.

I'm working on implementing the new questions. Like you mentioned, we don't have answers for some of them yet but we should soon. 

Quick question about one of the new questions, "What are these 20 different climate models?": 
This information is covered comprehesively on MACA's website: http://maca.northwestknowledge.net/GCMs.php
Would it be OK to link to it or maybe I should iframe it? Thoughts?

**Comment by hadinon** [comment]
OK, update on this.... I decided to include some information about the 20 different GCMs and then link to more information. I have a few placeholders in there with "coming soon". :smile: 

I believe I covered all your suggestions but I need to go through it one more time with a fine tooth comb. Let me know if you see anything that looks off. Otherwise, I will copy it over to the other server and we can add a link to it on the DSS.

**Comment by daviswx** [comment]
Regarding the GCMs, I like the added section with details about them.  You might also add a statement like "These models are run by universities and research institutions around the world.  They all differ slightly in their resolutions, the components of the earth and atmosphere that are modeled, and the physics and chemistry schemes implemented in them."

Once you get the rest of the page updated, I can look over it in more detail as well.

**Comment by hadinon** [comment]
OK, everything is ready for your review. The only sections which I am unable to update right now are:
- model error section under "Do global climate models show agreement with historical observations?" (error is still being computed)
- content under "Is the spread of historical conditions wider than the possibilities painted by future model output?" (historical variability is still being calculated)
- content under "Which models generate the productivity outputs?" (need to ask the modelers for a brief write-up)

Once your review is finished and updates are made, additional action items include: 
- migrating content to DSS template
- adding a link to this MACA / DSS FAQ on the Climate Data tab of the DSS Introduction page

Side note: Katherine (MACA postdoc) is reviewing the original MACA FAQ website (created for PINEMAP modelers). Once things are finalized with this one, I can ask her to review this one, too.

**Comment by daviswx** [comment]
Here are some suggestions:

**General**
- Change all references to "GCM's" to "GCMs" and "RCP's" to "RCPs" (with no apostrophes)

**What is MACA? How can I get more information about this method?**
- Possible re-wording for the first sentence: "The future climate data that forms the basis for PINEMAP DSS tools comes from the Multivariate Adaptive Constructed Analogs (MACA) dataset."
- Later in that paragraph, I wonder about the wording "This dataset was created by researchers...".  Maybe say "assembled" rather than "created" because the GCM data itself was created by other groups.
- After that, I would remove the link to the 20 global climate models since you talk about those in more detail (and include that link) in the next section.
- In the second paragraph, I wonder about the wording "computer-based models".  I would either say "computer models" or "computer-driven models".
- Even though you link to an explanation of statistical downscaling, I'm not sure whether everyone will click that link, and even if they do, that explanation may not be completely clear if the user isn't familiar with modeling terminology.  Here's an idea for an explanation you could put in the text:
  - In order to simulate the climate over such long time periods -- out to the end of the century -- and a global scale, these models run at a low spatial resolution that, by itself, would make local assessments challenging.  However, using a process called statistical downscaling, this global output is translated to local and regional projections using statistical relationships between climate variables.
- Later in that paragraph, for the statement in parentheses, either say "e.g.," at the beginning or "etc." at the end, but not both.
- In the next paragraph, the second sentence is a bit of a run-on.  Maybe you could say "Note that the two RCPs are indications of _future_ emissions, so they are not available for the climate model baseline period."
- As part of this same paragraph about MACA being a state-of-the-art method, I wonder if it might be worthwhile to include a statement about ways the data is being used.  Do we know of any other projects (either at our office or elsewhere) that are using MACA?  If so, we could say something like "Because of these key features, MACA data is being used by researchers in forestry, ecology, biology, and other fields that are sensitive to the climate."

**What are the 20 different global climate models included in the multi-model mean?**
- I wonder if the title of this section might be confusing since it could imply that the 20 GCMs are _only_ used in the middle maps.  Instead, what if you said something like "included in the MACA dataset"?

**How are the future Representative Concentration Pathways (RCP's) created?**
- In the first sentence, I wonder about the word "indications".  To me, that means the RCPs are giving us an indication of what the future emissions will be like.  Rather, I'd say that they are "scenarios depicting future emissions based on several assumptions...".
- In the next sentence, you use "range"/"ranging" twice.  I'd suggest getting rid of "ranging".

**How are future changes and averages in the PINEMAP DSS calculated?**
- In the second sentence, change the wording so it doesn't sound like an instruction.  Here's a suggestion: "This is computed for each RCP and for each GCM."
- Very minor suggestion, but in the final paragraph, use the "plus-minus" symbol (±, or HTML code `&plusmn;`) instead of "+/-".

**What does MACA climate model baseline data (1950-2005) represent?**
- I wonder if DSS users will understand what "free-form forecasts" mean?  Here's a way to word this sentence that might remove any uncertainty: "Instead, they are forecasts from the GCMs run over this period with the same model physics and configurations."
- The last paragraph in this section mentions that the GCMs don't use either of the RCPs for the baseline period, but we never mention what emissions scenario they _do_ use (and frankly, I'm not sure myself!).  If we can find this, maybe we could put it in the final paragraph or put it with my suggested sentence above (e.g., "...same model physics and configurations, but with an emissions scenario representative of past conditions").

**Which observed climate dataset was used for the PINEMAP DSS?**
- This section still sounds like it was written for modelers, not DSS users.  (I'm not sure if you made any changes from the modelers' version.)  Here's an idea for new wording:
  - Although we can't yet tell how accurate the GCMs are for the future time slices, we can assess their performance over the historical period by comparing the model baseline data with historical observations.  In our analysis, the UofIdaho MetData, which spans from 1979-2014, provided these historical observations.  This dataset was used because it provides the highest spatial resolution and all the critical variables and time steps needed.

**Do global climate models show agreement with historical observations (which gives confidence in future data)?**
- I would get rid of the part in parentheses in the title and include it in the text somewhere instead.
- Also get rid of the parenthetical statement "(which is the observed dataset used for the PINEMAP DSS)" since that was mentioned in the previous section.
- No explanation is given for the figure, and I'm guessing most users will read "PDF" and assume that's the file format the results were provided in or something like that.  We probably don't need anything too detailed, but it would be nice to have a basic explanation.  Also, since the MIROC5 sticks out like such a sore thumb, I feel like we need to have an explanation about why it's so bad, or why it's so different from everything else.  Do we know this or have an idea?

For now, I'd comment out the two sections "Is the spread of historical conditions wider than the possibilities painted by future model output?" and "Which models generate the productivity outputs?" and add them back in once we have those answers ready.

Once we're happy with the text, it should be really easy to copy over the code and link it in.

**Comment by hadinon** [comment]
All of your changes (as well as a few of my own) have been implemented. 

A note has been added to the "Do global climate models show agreement with historical observations?" about model error not being displayed yet within the DSS. Also, these two FAQs have been commented out for now based on your suggestion: "Is the spread of historical conditions wider than the possibilities painted by future model output?" and "Which models generate the productivity outputs?"

So, unless you want to quickly look it over one more time, the content is ready to migrate over to the DSS template. Then, maybe add a link to it from the Climate Data tab of the DSS Intro page on the dev site?

Lastly, I am going to update the other FAQ site (for modelers) based on some of your suggestions and then I will send a reminder email to Katherine to review the original FAQ and ask her if she would like to review the DSS one, too.

Anything else you can think of?

**Comment by daviswx** [comment]
Just one small wording question: In the first section title, is "method" the correct word to describe MACA?  Or should it be something like "dataset" instead?

I migrated the FAQ to the dev site:
http://sco-web1.meas.ncsu.edu/pinemap_dev/index.php?tool=faq

One small thing I changed was shrinking the PDF image and aligning it to the right of the text.  I realized that I couldn't fit the image and the explanation on my screen at the same time, and I thought it was probably best if you could see them both.

**Comment by hadinon** [comment]
Yep, agreed about the small wording change. It has been fixed. I also changed a few other minor wordings here and there. Can you copy over the script again to the dev site? 

Also, yes, the new PDF size and placement looks good to me. Thanks!

**Comment by daviswx** [comment]
Yes it's been copied over.

**Comment by hadinon** [comment]
Awesome, thanks. One minor change: in the PDF section, can you change it to say, "see figure on the right" instead of "below"?

**Comment by hadinon** [comment]
Now that all those changes have been made, I wanted to make a quick action items list (to be completed post-break!):
- Add information about model error once those calculations have finished
- Add answers to "Is the spread of historical conditions wider than the possibilities painted by future model output?" and "Which models generate the productivity outputs?"
- Once the new landing page is created with new menus, add a link to this FAQ DSS site 
- Implement feedback from Katherine

Please feel free to comment with any other items.

**Comment by daviswx** [comment]
Post-break update:
- Now that some of the historical standard deviations have been generated, I suspect we will soon be able to answer the question about historical variability vs. future projections.
- I have added a link to the DSS MACA FAQ page in the "About" menu.
- Are we still waiting on feedback from Katherine?

**Comment by hadinon** [comment]
- Yes, I will plan to revisit some of these questions once the historical standard deviation maps are finished. 
- Link looks good! 
- Yes, I'll send a reminder email to Katherine :smiley:

**Comment by hadinon** [comment]
I probably need to revisit the action items here soon. In the meantime, at our meeting with Ryan yesterday, we talked about adding a FAQ about how to reference/cite the DSS. What do we want a citation to look like?  Here's how some comparable tools handle a citation FAQ:
- Climate Wizard: http://www.climatewizard.org/FAQ.html#_Can_I_select_2
- USGS GeoData Portal has a point of contact and distribution information that pops up when you click on each product here: http://cida.usgs.gov/gdp/

IMO, Climate Wizard's approach fits better with the DSS. We don't have a publication to link to yet so I like the idea of using the contents similar to what Climate Wizard has on their "About Us" page, such as: "The data and information presented in this web site are subject to the conditions stated below.  Data may be freely reproduced and distributed for non-commercial purposes. When referring to the data, the source must be clearly and prominently stated, and include at a minimum, our name, URL.  Use the citations for appropriate data in the manner below: [insert list of data sources including MACA, METDATA, and maybe regional modeling outputs? we could start with MACA and METDATA.]"

I envision providing an example citation in APA format, like this: 
PINEMAP Decision Support System. (2015, December). Retrieved [insert today's date], from http://climate.ncsu.edu/pinemap

Thoughts?

**Comment by daviswx** [comment]
That sounds fine to me.  I actually like their full ["About Us" page](http://www.climatewizard.org/AboutUs.html), so I wonder if we might want to create something similar.  It might have:
- A short overview of the data and products in the DSS
- Info. about the DSS developers (similar to what we have at the top of the DSS Development page)
- Info. and any citations about data or map layers used, like MACA, UofI METDATA, specific ecological model output, etc.
- Acknowledgement of our funding sources (of course, it's also at the bottom of the DSS)

I'm thinking we could rename the DSS Development page to DSS Change Log, and move the development info onto this separate About the DSS page.

**Comment by hadinon** [comment]
PINEMAP Decision Support System version X.X. (2015, December). Retrieved [insert today's date], from http://climate.ncsu.edu/pinemap

**Comment by daviswx** [comment]
In today's meeting, we discussed adapting the DSS Development page to include the same information as the proposed About the DSS page, in addition to keeping the change log (in a smaller, scrollable box).

This page would also include the sample citation (listed above) for the DSS, automatically filled in with the current date and the latest DSS version.

Should this citation be tool-specific?  And if so, should it be included somewhere on each tool?

**Comment by hadinon** [comment]
Below is the guidance that I sent to the person inquiring about a DSS citation:

"Here is an example citation for the DSS in APA format:

PINEMAP Decision Support System, version X.X. (2015, December). Retrieved [insert date here], from http://climate.ncsu.edu/pinemap/

Please feel free to adjust the order of wording and/or punctuation within the citation if you are using a format other than APA (e.g., MLA, Chicago). 

In order to fill in the version number, please consult this page: http://climate.ncsu.edu/pinemap/index.php?tool=dev
The latest version is 1.3 so you'll likely use that but I wasn't sure if you had accessed/used the data prior to that date.

Also, I just wanted to give a heads up that the DSS URL may be changing soon to something simpler, such as pinemapdss.com."

**Comment by hadinon** [comment]
I don't think the citation would need to be tool specific, but we could carve out a section at the bottom of each tool if we did want to go that route. Thoughts? 

Looking over my notes from the meeting, I also wrote down that we could move information about data sources/links on the DSS Intro --> Climate Data tab to the DSS development section. I'll post more about what to keep within the Climate Data tab in issue #93.

**Comment by daviswx** [comment]
I can see a benefit for putting a citation at the bottom of each tool, beneath the Other Resources section.  Once we finalize the citation format and the updated DSS Development/About the DSS page, I will work on adding that in.
